webapps-schema/src/main/java/io/camunda/webapps/schema/entities/clustervariable/ClusterVariableEntity.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps.schema.entities.clustervariable;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.camunda.webapps.schema.entities.ExporterEntity;
+import java.util.Arrays;
+import java.util.Objects;
+
+public class ClusterVariableEntity implements ExporterEntity<ClusterVariableEntity> {
+
+  private String clusterVariableId;
+  private String name;
+  private String value;
+  private String fullValue;
+  private boolean isPreview;
+  private String scope;
```

Comment:
🔧 I understand that it could be a String, but to be consistent I would suggest making it out of it an enum to avoid issues.